src/NorthwindWebsite/Business/Services/Implementations/ProductService.cs

```diff
@@ -51,6 +51,9 @@ public async Task<ProductsDto> GetProducts()
         return productsListDto;
     }
 
+    public async Task<Product> GetProduct(int id) =>
+        await _productRepository.Get(id, true);
```

Comment:
Don't forget about adding names for parameters to make `true` value purpose more obvious!